### DISCUSSION TITLE
cannot run sftp on version V8.9.1.0 on windows server 2012

### DISCUSSION THREAD

**Discussion by bogdan-atl** [PowerShell/Win32-OpenSSH#2001] [Q&A]
For some reason, the connection itself breaks, without error.,identical problem on windows 7

sftp -P 2200 -i id_ecdsa_sk_yubikey4 bogdan@192.168.0.93
Enter passphrase for key 'id_ecdsa_sk_yubikey4': 
Confirm user presence for key ECDSA-SK SHA256:x5swo2zbVQCcG6bcгc42VevBrrjjDgIBQK/eSravAkU
User presence confirmed
Connection closed.  
Connection closed

**Comment by maertendMSFT** [comment]
Created an issue thread instead of a discussion: https://github.com/PowerShell/Win32-OpenSSH/issues/2007
